### DISCUSSION TITLE
Deleting redis queue

### DISCUSSION THREAD

**Discussion by wolfmatt21** [Security-Onion-Solutions/securityonion#5816] [Unsupported Versions] [answered]
HI,

is there any way to clear a redis queue on a heavy node? have backed up events that need clearing.

**Comment by dougburks** [comment, marked as answer]
Yes, you should be able to use `redis-cli` via `docker exec` to clear the redis queue. Take a look at `/usr/sbin/so-redis-count` for an example of how to run `redis-cli` via `docker exec`:
https://github.com/Security-Onion-Solutions/securityonion/blob/master/salt/common/tools/sbin/so-redis-count#L20

**Comment by Humoud** [comment]
Use at your own risk
`docker exec so-redis redis-cli del logstash:unparsed`
